src/main/java/core/basesyntax/serviceimpl/OperationStrategyImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private Map<FruitTransaction.Operation, OperationHandler> operationHandlersMap;
+
+    public OperationStrategyImpl(Map<FruitTransaction.Operation,
+            OperationHandler> operationHandlersMap) {
+        this.operationHandlersMap = operationHandlersMap;
+    }
+
+    @Override
+    public OperationHandler get(FruitTransaction.Operation operation) {
+
+        return operationHandlersMap.get(operation);
+
+    }
+
```

Comment:
Remove the last empty line in all classes/interfaces/methods where it is present